QUICKSTART.md

```diff
@@ -0,0 +1,361 @@
+# **KCP-Edge** Quickstart
+
+Table of contents:
+
+- [1. Install **KCP-Edge** pre-requisites](#1-install-kcp-edge-pre-requisites)
+  - [a. `kubectl`](#a-kubectl)
+  - [b. `kcp`](#b-kcp)
+- [2. Install and run **KCP-Edge**](#2-install-and-run-kcp-edge)
+- [3. Create a **KCP-Edge** Inventory Management Workspace (IMW) and Workload Management Workspace (WMW)](#3-create-a-kcp-edge-inventory-management-workspace-imw)
```

Comment:
I thought the plan for the quickstart is to use a script that wraps up steps 1, 2, and optionally 3a and optionally 3b.

Reply:
Could you review the newly update USER-GUIDE.md? To address your comments above required a significant change of the structure outline in that table of contents

Reply:
We still can revise that quickstart.md to wrap up steps 1 and 2 indeed.